### DISCUSSION TITLE
Access zoomed geometry values?

### DISCUSSION THREAD

**Discussion by zach-tubb** [uber/nebula.gl#793] [Q&A]
Is there a way to access zoomed geometry values currently? If I draw a polygon feature, then change the zoom level of the view, the polygon is zoomed as expected. But when observing the underlying feature geometry, those values remain unchanged. I have a use case that necessitates having access to the zoomed feature geometry values. It would seem nebula is tracking those updated values somewhere but I can't see to find where they are/how to access them.

**Comment by ibgreen** [comment]
In a 3D environment, talking about "zoomed values" isn't very precise. Are you looking for screen coordinates, rather than lat/lngs? If so, deck.gl has project functions that allow you to calculate this on the CPU for your data based on current viewport.

**Reply by zach-tubb** [reply]
Hi @ibgreen! Yes sorry, I am new to the GIS domain. I think screen coordinates is what I am looking for. When I zoom/scale the parent deck.gl view I am trying to see what the new screen(?) geometry of the nebula features are in that scaled view.

**Reply by ibgreen** [reply]
Start here https://deck.gl/docs/api-reference/core/layer#layer-projection-methods. Open discussion in deck.gl if you have more questions (or join deck.gl slack).
